scripts/des_y3_sne_ia/download_data.py

```diff
@@ -0,0 +1,43 @@
+import os
+import argparse
+import requests
+import tarfile
+import requests
+import os
+def download_text_file(url, file_path):
+    response = requests.get(url)
+    if response.status_code == 200:
+        with open(file_path, 'w') as file:
+            file.write(response.text)
+    else:
+        print(f"Failed to download text file. Status code: {response.status_code}")
+
+def read_text_file(file_path):
+    with open(file_path, 'r') as file:
+        contents = file.read()
+        return contents
+    
+
+
+
+def main(args):
+
+    # Construct the URL to download 
+    url = 'https://raw.githubusercontent.com/PantheonPlusSH0ES/DataRelease/main/Pantheon%2B_Data/1_DATA/photometry/DES3YR_DES_COMBINED_TEXT/'
+    
+    #name of files
+    names=read_text_file('DES3YR_DES_LIST.txt').split('\n')[:-1]
+
+    if not os.path.exists(args.destination_path+'/des_y3_sne_ia/'):
```

Comment:
nit: more canonical to use `os.path.join(orig_path, subdir_name)` or `from pathlib import Path; Path(orig_path) / "subdir_name"` to create new paths

Reply:
Ah my mistake I should have changed this after experimentation. I changed the code and download_data.py for all five surveys now uses os.path.join